mvt/feature.go

```diff
@@ -2,40 +2,27 @@ package mvt
 
 import (
 	"context"
-	"encoding/json"
 	"fmt"
-	"os"
 
-	"github.com/go-spatial/geom"
 	"github.com/go-spatial/geom/encoding/wkt"
 	"github.com/go-spatial/tegola"
-	"github.com/go-spatial/tegola/basic"
 	"github.com/go-spatial/tegola/internal/convert"
 	"github.com/go-spatial/tegola/internal/log"
-	"github.com/go-spatial/tegola/maths"
-	"github.com/go-spatial/tegola/maths/simplify"
-	"github.com/go-spatial/tegola/maths/validate"
 	vectorTile "github.com/go-spatial/tegola/mvt/vector_tile"
 )
 
 var (
 	ErrNilFeature          = fmt.Errorf("feature is nil")
 	ErrUnknownGeometryType = fmt.Errorf("unknown geometry type")
-	ErrNilGeometryType     = fmt.Errorf("nil geometry")
+	ErrNilGeometryType     = fmt.Errorf("geometry is nil")
 )
 
-// TODO: Need to put in validation for the Geometry, at current the system
-// does not check to make sure that the geometry is following the rules as
-// laid out by the spec. It just assumes the user is good.
-
 // Feature describes a feature of a Layer. A layer will contain multiple features
 // each of which has a geometry describing the interesting thing, and the metadata
 // associated with it.
 type Feature struct {
-	ID   *uint64
-	Tags map[string]interface{}
-	// does not support the collection geometry, for this you have to create a feature for each
-	// geometry in the collection.
```

Comment:
MVT does not support Geometry Collections and the `encodeGeoemtry` method returns an `ErrUnknownGeometryType` error if it's passed an unsupported type. I didn't see a reason to keep the comment around, though it may be helpful.